IOSum/src/SortedLetters.java

```diff
@@ -0,0 +1,15 @@
+import java.util.Arrays;
+import java.util.Comparator;
+
+public class SortedLetters {
+    public static void main(String[] args) {
+        String [] a = {"aaa","bbbb","a","bb"};
```

Comment:
what is the task you are trying to solve here? Do you want to sort a list of strings?